examples/multipart/sink.cxx

```diff
@@ -24,6 +24,8 @@ struct Sink : fair::mq::Device
     {
         LOG(info) << "Received message with " << parts.Size() << " parts";
 
+        assert(parts.Size() == 7);
```

Comment:
hm, this will be compiled away if `NDEBUG` is set, won't it?

Reply:
true, I'll change it.